src/handlers/project_goals.rs

```diff
@@ -45,14 +45,30 @@ impl Job for ProjectGoalsUpdateJob {
 
 /// Returns true if the user with the given github id is allowed to ping all group people
 /// and do other "project group adminstrative" tasks.
-pub async fn check_project_goal_acl(_gh: &GithubClient, gh_id: u64) -> anyhow::Result<bool> {
-    /// Github ID of the user allowed to ping all group people.
-    ///
-    /// FIXME: We should create a team for the person/people managing the goals program
-    /// and check that the zulip person is on it, but I'm too
-    const GOAL_OWNER_GH_ID: u64 = 155238; // nikomatsakis
-
-    Ok(gh_id == GOAL_OWNER_GH_ID)
+pub async fn check_project_goal_acl(gh: &GithubClient, gh_id: u64) -> anyhow::Result<bool> {
+    const GOALS_TEAM: &str = "goals";
```

Comment:
To be honest, not 100% sure if this is the way we pass team names down to `get_team`

Reply:
goals show up here https://team-api.infra.rust-lang.org/v1/teams.json with the right group of members, so it should be fine